.github/workflows/integration-tests.yaml

```
@@ -1,41 +1,56 @@
       - dev  # Adjust to your default branch (e.g., 'master', 'develop')
 
 jobs:
-  dotnet-integration-tests:
-    runs-on: ubuntu-latest  # Use ubuntu-latest; switch to windows-latest if needed
+  integration-tests:
+    runs-on: windows-latest
+    timeout-minutes: 10
 
     steps:
-    # Checkout the repository code
     - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)